composer.json

```diff
@@ -34,5 +35,8 @@
       "Lullabot\\Mpx\\Tests\\": "tests/src/",
       "Lullabot\\Mpx\\Command\\": "command/src/"
     }
+  },
+  "extra": {
+    "composer-version": "^1.6.3"
```

Comment:
This is for `composer-gavel` support and will be ignored if the plugin isn't installed.